### PR TITLE
Swag shop user story #2, #13 & #19: Landing page - "Products We Love", Hero, "word of the day"

### PR DESCRIPTION
**Pull request by prinzdezibel**
Notice: 
- This PR depends on another [PR](https://github.com/reactioncommerce/reaction-swag-shop/pull/4), which is implicitly included in this PR.
- Also,  this PR depends on another PR in Reaction core: https://github.com/reactioncommerce/reaction/pull/3383

This PR resolves issues #2, #13 and #19.

    (feat): As an admin, allow to specifify a new text property `featuredProductLabel`
    for products. All products that have specified this property will be
    displayed in section "Products We Love". The propertie's value will be
    displayed as turquoise label in the upper left corner of the product
    image.

### CODE REVIEW

File: server/methods.js [hunk 1]
```diff
@@ -116,7 +135,7 @@ methods.importProductImages = function () {
       const productId = product._id;
       if (!Media.findOne({ "metadata.productId": productId })) {
         const shopId = product.shopId;
-        const filepath = "custom/images/" + productId + ".jpg";
+        const filepath = "plugins/reaction-swag-shop/images/" + productId + ".jpg";
```
Comment (prinzdezibel) [inline server/methods.js]: This will only work, if `reaction-cli` provides the new build step: "Asset provisioning"

**Comment by prinzdezibel** [comment]
@zenweasel: PR should satisfy all acceptance criteria.

**Comment by prinzdezibel** [comment]
Notice: This PR depends on another PR in core: https://github.com/reactioncommerce/reaction/pull/3383

**Comment by prinzdezibel** [comment]
@zenweasel This branch is called michael-issue-2. Nevertheless, somehow, #13 and #19 sneaked in as well.

**Comment by zenweasel** [comment]
@prinzdezibel You need to resolve conflicts before I can test

**Pull request merged**